src/java/htsjdk/variant/vcf/AbstractVCFCodec.java

```diff
@@ -752,13 +753,19 @@ private final static boolean isVCFStream(final InputStream stream, final String
         return new LazyGenotypesContext.LazyData(genotypes, header.getSampleNamesInOrder(), header.getSampleNameToOffset());
     }
 
-    private final String[] INT_DECODE_ARRAY = new String[10000];
+
     private final int[] decodeInts(final String string) {
-        final int nValues = ParsingUtils.split(string, INT_DECODE_ARRAY, ',');
+        // find number of CSV items to set array size
+        int numItems = string.split(",").length;
```

Comment:
Unfortunately, no. I wish maven was that smart and would import it. In a vanilla build system, you could do that and just add it to the classpath. I might've found the answer, the pom file does not have the artifactId for the jar containing this functionality, only commons-jexl. It looks like I need commons-lang (https://commons.apache.org/proper/commons-lang/)